pkg/validate/validate_test.go

```diff
@@ -563,3 +563,29 @@ func TestURI(t *testing.T) {
 		})
 	}
 }
+
+func TestUnicastMAC(t *testing.T) {
+	cases := []struct {
+		addr    string
+		unicast bool
+	}{
+		{
```

Comment:
Since the validation now parses the MAC, you could add a test case for a malformed address.

Reply:
Good call, updated the test cases